service.sh

```diff
@@ -17,8 +17,10 @@ beertime() {
         printf '%s' $(( $(date -u -d "$later" +%s) - $(date -u -d "$now" +%s) ))
       }
 
-      local now=$(date -d 'now' '+%F %T %Z')
-      local later=$(date -d "today $1:00:00" '+%F %T %Z')
+      local now=
+      now=$(date -d 'now' '+%F %T %Z')
+      local later=
+      later=$(date -d "today $1:00:00" '+%F %T %Z')
       local hrsUntil=$(( $(diffTime) / 60 / 60 ))
```

Comment:
What's the reason for not applying the same predefined variables for `hrsUntil` and `minUntil`?